### PR TITLE
[Action Needed] Update CRUD files for newly added properties

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

**Comment by RuiHou105** [comment]
**Roadmap workitem:**

Feature 12535: [Devices and apps][Cloud PC] Update CRUD files for added properties

**Comment by RuiHou105** [comment]
@isvargasmsft  @JarbasHorst  One of the check failed. I am not sure the cause. Could you please take a look? Thanks ahead.

File: api-reference/beta/api/cloudpc-get.md [hunk 1]
```diff
@@ -92,9 +94,9 @@ GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPCs/{
 ---
 
 
-### Response
+#### Response
 
-**Note:** The response object shown here might be shortened for readability.
+**Note:**
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:99]: If you don't need the note, I recommend you to remove the entire line.

File: api-reference/beta/api/cloudpc-get.md [hunk 2]
```diff
@@ -137,3 +141,53 @@ Content-Type: application/json
...
+#### Request
+
+<!-- {
+  "blockType": "request",
+  "name": "get_cloudpc"
+}
+-->
+
+``` http
+GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPCs/{id}?$select=id,displayName,,imageDisplayName,lastModifiedDateTime,lastRemoteActionResult,lastLoginResult
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:156]: Please, remove the duplicated comma after "displayName".

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md [hunk 3]
```diff
@@ -83,6 +84,7 @@ Content-Type: application/json
 {
   "@odata.type": "#microsoft.graph.cloudPcOnPremisesConnection",
   "displayName": "Display Name value",
+  "type": "Azure Active Directory join type",
```
Comment (JarbasHorst) [inline api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:87]: For clarity, please provide a "real" example here. Same for line 139.

Ideally, the other properties in these two examples (Request and Response) should also contain "real" data.

File: api-reference/beta/api/cloudpc-get.md (hunk 2 above)
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:156]: For clarity, please replace the placeholder {id} with a "real" example. Same for line 72.

File: api-reference/beta/api/cloudpc-get.md [hunk 4]
```diff
@@ -137,3 +141,53 @@ Content-Type: application/json
...
 }
 ```
+
+### Example 2: Get the selected properties of a Cloud PC
+
+#### Request
+
+<!-- {
+  "blockType": "request",
+  "name": "get_cloudpc"
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:151]: Hi @RuiHou105, example 1 and 2 have the same name "get_cloudpc". This is the reason why we are getting that error in the API Doctor tool. Please, make sure to add unique names to your examples. Suggested revision:

- (line 67) "name": "get_cloudpc_default_properties"
- (line 151) "name": "get_cloudpc_selected_properties"

File: api-reference/beta/api/cloudpc-get.md (hunk 2 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:160]: Updated.

File: api-reference/beta/api/cloudpc-get.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:103]: Removed. Thanks.

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md (hunk 3 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:87]: Updated the examples to user real values.

File: api-reference/beta/api/cloudpc-get.md (hunk 2 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:160]: Updated.

File: api-reference/beta/api/cloudpc-get.md (hunk 4 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:155]: @JarbasHorst  Thanks for your help! All comments resolved. Could you please take a second review then you got time? Thanks ahead!

File: api-reference/beta/api/cloudpc-get.md [hunk 5]
```diff
@@ -96,13 +98,12 @@ GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPCs/{
 ---
 
 
-### Response
+#### Response
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:102]: For consistency, please add the following boilerplate text here:

- The following is an example of the response.

File: api-reference/beta/api/cloudpc-get.md [hunk 6]
```diff
@@ -141,3 +144,52 @@ Content-Type: application/json
...
+  "name": "get_cloudpc_selected_properties"
+}
+-->
+
+``` http
+GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPCs/40cee9d2-03fb-4066-8d35-dbdf2875c33f?$select=id,displayName,imageDisplayName,lastModifiedDateTime,lastRemoteActionResult,lastLoginResult
+```
+
+#### Response
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:163]: For consistency, please add the following boilerplate text here:

- The following is an example of the response.

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md [hunk 7]
```diff
@@ -122,7 +124,6 @@ Content-Type: application/json
 
 ### Response
 
-**Note:** Here is an example of the response. The response object shown here might be shortened for readability.
```
Comment (JarbasHorst) [inline api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:125]: 1) For consistency, please add the following boilerplate text to line 72:

- The following is an example of a request.

2) For consistency, please add the following boilerplate text here:

- The following is an example of the response.

File: api-reference/beta/api/cloudpc-get.md [hunk 8]
```diff
@@ -141,3 +144,52 @@ Content-Type: application/json
     "gracePeriodEndDateTime": "2020-11-010T20:00:34Z"
 }
 ```
+
+### Example 2: Get the selected properties of a Cloud PC
+
+#### Request
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:151]: For consistency, please add the following boilerplate text here:

- The following is an example of a request.

File: api-reference/beta/api/cloudpc-get.md [hunk 9]
```diff
@@ -56,18 +56,20 @@ If successful, this method returns a `200 OK` response code and a [cloudPC](../r
 
 ## Examples
 
-### Request
+### Example 1: Get the default properties of a Cloud PC
+
+#### Request
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpc-get.md:62]: For consistency, please add the following boilerplate text here:

- The following is an example of a request.

File: api-reference/beta/api/cloudpc-get.md (hunk 5 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:102]: Added.

File: api-reference/beta/api/cloudpc-get.md (hunk 6 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:163]: Updated. Thanks.

File: api-reference/beta/api/cloudpc-get.md (hunk 8 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:151]: Added.

File: api-reference/beta/api/cloudpc-get.md (hunk 9 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpc-get.md:62]: Added.

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md (hunk 7 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:125]: Added.
@JarbasHorst  Thanks to your comments. All comments resolved. Could you please take a second look when you got time? Thanks!

**Comment by JarbasHorst** [comment]
@Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**